introduction.md

```diff
@@ -3,3 +3,12 @@
 This is an introduction for the dojo_rules repository, as required by the Code School course "Mastering GitHub".
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?